### ISSUE TITLE
Non launch of any windows os

### ISSUE THREAD

**Issue by 66schoollane** [quickemu-project/quickemu#649]
#### Expected behaviour
when opening quickemu GUI & downloaded the following i'm
unable to launch any windows OS, get a new box opens connect to spice 
paste board image of win os loading
https://pasteboard.co/6QQYmJ8WmVyl.png

all linux OS 100% works and launches automatically 
paste board image of a linux os 
https://pasteboard.co/x8qRZRfiHapw.png

not sure how to fix this issue

#### Actual behaviour

Describe the actual behaviour.

#### Steps to reproduce the behaviour

install as instuctions.

#### Quickemu output

Run `quickemu` or `quickget` and paste the output here.

#### Linux Distribution & Kernel

Run the following and paste the output.

```bash
lsb_release --all
uname -a
```
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.10
Release:	22.10
Codename:	kinetic

Linux blackbox 5.19.0-31-generic #32-Ubuntu SMP PREEMPT_DYNAMIC Fri Jan 20 15:20:08 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

**Comment by 5Dev24** [comment]
Is this specifically an issue with quickemu or within quickgui? You may want to put your issue on [quickgui](https://github.com/quickemu-project/quickgui) first so they can see if it's an issue on their end.

**Comment by hoo761** [comment]
@66schoollane

I had the same issue and was able to fix it.

It is noted in the [README](https://github.com/quickemu-project/quickemu#windows-10--11-guests):

> `quickget` can not download [Windows10](https://www.microsoft.com/software-download/windows10) and [Windows 11](https://www.microsoft.com/software-download/windows11) automatically, but does automatically create an optimised virtual machine configuration that you can just add an Windows .iso image to.

You will need to download the Windows ISO that you want and move it to the path that your `.conf` file specifics. You will also need to change the name of the downloaded iso to match what your `.conf` specifies. `quickgui` places the `windows-11.conf` and the `windows-11` folder in your home directory.

In the case of the default `.conf` for Windows 11 it is `windows-11/windows-11.iso`

```
#!/usr/bin/quickemu --vm
guest_os="windows"
disk_img="windows-11/disk.qcow2"
iso="windows-11/windows-11.iso"
fixed_iso="windows-11/virtio-win.iso"
tpm="on"
secureboot="off"

```

After I did that, the machine booted into the Windows setup. After that was finished,  it boots into the OS and you should be good to go!

**Comment by flexiondotorg** [comment]
Fixed via #836
